tests/PySys/mqtt_port/mqtt_port_change_connection_works/run.py

```diff
@@ -0,0 +1,132 @@
+import sys
+
+sys.path.append("environments")
+from environment_roundtrip_c8y import Environment_roundtrip_c8y
+
+from pysys.basetest import BaseTest
+
+import time
+
+"""
+Validate changing the mqtt port using the tedge command
+
+Given a configured system, that is configured with certificate created and registered in a cloud
+When the thin edge device is disconnected from cloud, sudo tedge disconnect c8y/az
+When mqtt.port is set using tedge with sudo
+When the thin edge device is connected to cloud, sudo tedge connect c8y/az
+Now validate the services that use the mqtt port
+   Validate tedge config set mqtt.port
+   Validate tedge mqtt pub/sub
+   Validate tedge connect c8y/az --test
+   Validate tedge_mapper status
+
+"""
+
+class MqttPortChangeConnectionWorks(Environment_roundtrip_c8y):
```

Comment:
Not sure if it makes sense here to use the c8y environment. It connects automatically before the test is started

Reply:
I feel I dont need any environments